cornice/pyramidhook.py

```diff
@@ -219,7 +219,8 @@ def register_service_views(config, service):
         predicate_definitions = _pop_complex_predicates(args)
 
         if predicate_definitions:
-            for predicate_list in predicate_definitions:
+            extra_empty_ctype = [({'kind': 'content_type', 'value': ''},)]
```

Comment:
Ctype [refers to something different](https://docs.python.org/2/library/ctypes.html) in the python context.